### ISSUE TITLE
Pool efforts with vanjacosic

### ISSUE THREAD

**Issue by mixonic** [relekang/django-community-versions#3]
I think there is another fork that happened about the same time as yours:

https://github.com/vanjacosic/django-community-versions

Y'all should get together. /cc @vanjacosic. I'm unsure how easily we can share improvements across the projects, but keeping the forks focused is a good first step ;-)

**Comment by relekang** [comment]
Yeah, we both found out about both efforts through twitter, later on. I think this one will be deleted later today.
